### PR TITLE
User can view their entries

### PR DESCRIPTION
**Pull request by omidzed**
https://github.com/omidzed/code-journal-c1023/assets/136918229/ca26ee70-a29f-4954-a1b7-2f6a8683fa78

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,15 +23,18 @@
     <link rel="stylesheet" href="css/styles.css" />
   </head>
   <body>
-    <header>
-      <div class="container">
-        <div class="column-full"><h1 id="code-journal">Code Journal</h1></div>
-      </div>
+    <header class="container">
+      <h1 id="code-journal">Code Journal</h1>
+      <a class="entries-anchor">Entries</a>
```
Comment (shawnkost) [inline index.html:28]: Needs an `href="#"`

File: index.html [hunk 2]
```diff
@@ -46,16 +49,26 @@
...
+            <button>SAVE</button>
           </div>
         </form>
       </div>
-      <div data-view="entries" class="hidden"></div>
+
+      <div data-view="entries" class="view-container">
+        <div class="row entries-new">
+          <h2>Entries</h2>
+          <a class="new-anchor">NEW</a>
```
Comment (shawnkost) [inline index.html:67]: Needs an `href="#"`

File: js/main.js [hunk 3]
```diff
@@ -14,13 +27,88 @@ $entryForm.addEventListener('submit', submitHandler);
...
   const entry = {
-    entryId: data.nextEntryId++,
+    entryId: data.nextEntryId,
     title: $title.value,
     notes: $notes.value,
     photoUrl: $photoUrl.value,
   };
-  data.entries.unshift(entry);
+
+  data.entries.push(entry);
```
Comment (shawnkost) [inline js/main.js:38]: You should be adding to the beginning of the array rather than the end

File: js/main.js [hunk 4]
```diff
@@ -14,13 +27,88 @@ $entryForm.addEventListener('submit', submitHandler);
...
+
+  const $columnHalfOne = document.createElement('div');
+  $columnHalfOne.setAttribute('class', 'column-half');
+
+  const $columnHalfTwo = document.createElement('div');
+  $columnHalfTwo.setAttribute('class', 'column-half');
+
+  const $entryPhoto = document.createElement('img');
+  $entryPhoto.setAttribute('class', 'entry-photo');
+  $entryPhoto.setAttribute('src', entry.photoUrl);
```
Comment (shawnkost) [inline js/main.js:65]: Needs an `alt` attribute

**Review by shawnkost — CHANGES_REQUESTED** [review]
Looking pretty good! Just a few tweaks here

File: css/reset.css [hunk 5]
```diff
@@ -1,3 +1,4 @@
+/* stylelint-disable string-quotes */
```
Comment (shawnkost) [inline css/reset.css:1]: Don't add any lint rules without asking. We have our rules in place for a reason

**Review by shawnkost — CHANGES_REQUESTED** [review]
One more change here!

File: css/reset.css (hunk 5 above)
Comment (omidzed) [reply css/reset.css:1]: copy that.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**